src/Deposit.js

```diff
@@ -1,13 +1,16 @@
-import EventEmitter from "events"
+import { EventEmitter } from "events"
```

Comment:
This is filed under “wait, did that even work?”

Reply:
😆 It actually did!

Reply:
😅 the follow-up question is immediately: will it still work? <_<

Reply:
Fair point! :)